### PR TITLE
Specify required Ruby version

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

File: minitest-reporters.gemspec [hunk 1]
```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |s|
   s.summary     = %q{Create customizable Minitest output formats}
   s.description = %q{Death to haphazard monkey-patching! Extend Minitest through simple hooks.}
 
+  s.required_ruby_version = '>= 1.9.3'
```
Comment (os97673) [inline minitest-reporters.gemspec]: why 1.9.3?  Why not 1.8.7?  It looks like minitest doesn't restrict ruby version.

Comment (nicolasleger) [inline minitest-reporters.gemspec]: @os97673 The build is not tested on Ruby 1.8.7, we should add it to Travis to check build tests at least.

Comment (nicolasleger) [inline minitest-reporters.gemspec]: @os97673 I just added the test on Ruby 1.8.7 in the PR but tests are failing, do we want to fix code to support 1.8.7?

Comment (os97673) [inline minitest-reporters.gemspec]: No I don't.  Ok, I convinced ;)

**Comment by os97673** [comment]
Please remove the last commit and I will merge the PR.

**Comment by nicolasleger** [comment]
Done !

**Comment by os97673** [comment]
Thank you for the contribution.

**Pull request merged**